### ISSUE TITLE
Assertion violation at exists_forall/ef_skolemize.c:243

### ISSUE THREAD

**Issue by rainoftime** [SRI-CSL/yices2#258]
Hi, for the following formula
~~~~
(set-logic LIA)
(declare-const i6 Int)
(assert (exists ((q17 Bool) (q18 Int) (q19 Int)) (distinct (abs (mod i6 36)) 0)))
(check-sat)
~~~~

 Yices 59d786df2
~~~~
Unsupported term (abs (mod i6 36)) of kind 13
yices_smt2: exists_forall/ef_skolemize.c:243: ef_update_composite: Assertion `0' failed.
~~~~

**Comment by rainoftime** [comment]
Seems related to the support of the `abs` function

**Comment by aman-goel** [comment]
Fixed in ed563f0cf55c5476821ac371f658d13f88bdd800.
